crates/core/tedge_agent/src/restart_manager/config.rs

```diff
@@ -1,12 +1,14 @@
 use camino::Utf8PathBuf;
 use tedge_api::mqtt_topics::EntityTopicId;
+use tedge_config::SudoCommandBuilder;
 
 #[derive(Debug, Clone)]
 pub struct RestartManagerConfig {
     pub device_topic_id: EntityTopicId,
     pub tmp_dir: Utf8PathBuf,
     pub config_dir: Utf8PathBuf,
     pub state_dir: Utf8PathBuf,
+    pub sudo_wrapper: SudoCommandBuilder,
```

Comment:
[Minor] I have a preference for the name `sudo` as used by the plugin manager.

 ```suggestion
     pub sudo: SudoCommandBuilder,
```